docs/smart_scripts.md

```diff
@@ -31,11 +33,11 @@
 | [target_param2][10]   | int(10)     | unsigned   |     | NO   | 0       |       |               |
 | [target_param3][10]   | int(10)     | unsigned   |     | NO   | 0       |       |               |
 | [target_param4][10]   | int(10)     | unsigned   |     | NO   | 0       |       |               |
-| [target_x][10]        | float       |            |     | NO   | 0       |       |               |
-| [target_y][10]        | float       |            |     | NO   | 0       |       |               |
-| [target_z][10]        | float       |            |     | NO   | 0       |       |               |
-| [target_o][10]        | float       |            |     | NO   | 0       |       |               |
-| [comment][11]         | text        |            |     | NO   | NULL    |       | Event Comment |
+| [target_x][11]        | float       |            |     | NO   | 0       |       |               |
+| [target_y][12]        | float       |            |     | NO   | 0       |       |               |
+| [target_z][13]        | float       |            |     | NO   | 0       |       |               |
+| [target_o][14]        | float       |            |     | NO   | 0       |       |               |
+| [comment][15]         | text        |            |     | NO   |         |       | Event Comment |
```

Comment:
change back to 10 because \#target_x is not it's own topic and therefore the link will not work.

Also remove unused links under

Reply:
Okay, I'll change it now.

Reply:
Nice =)